src/radix_connect/well_known_client/models/dapp_definition.rs

```diff
@@ -1,23 +1,29 @@
 use crate::prelude::*;
 
-#[derive(Debug, Serialize, Deserialize, PartialEq)]
+/// Wrapper around the address of a dapp.
```

Comment:
Is this just a wrapper around a dapp definition ADDRESS? If it is, then we should delete this, we already have:

```rust
pub type DappDefinitionAddress = AccountAddress;
```

I did not notice at first, since this type was just named `DappDefinition`. If this is not that, but an actual structure, with more than one field, it should remain. But at this point it is not. So I vote, delete this and use the already existing `DappDefinitionAddress` type (alias)!